lib/Alien/Build/Plugin/Core/CleanInstall.pm

```diff
@@ -0,0 +1,53 @@
+package Alien::Build::Plugin::Core::CleanInstall;
+
+use strict;
+use warnings;
+use Alien::Build::Plugin;
+use Path::Tiny ();
+
+# ABSTRACT: Implementation for clean_install hook.
+# VERSION
+
+=head1 SYNOPSIS
+
+ use alienfile;
+ # already loaded
+
+=head1 DESCRIPTION
+
+This plugin implements the default C<clean_install> hook.
+You shouldn't use it directly.
+
+=head1 SEE ALSO
+
+L<Alien::Build>, L<Alien::Base::ModuleBuild>
+
+=cut
+
+sub init
+{
+  my($self, $meta) = @_;
+
+  $meta->default_hook(
+    clean_install => sub {
+      my($build) = @_;
+      my $root = Path::Tiny->new(
+        $build->runtime_prop->{prefix}
+      );
+      foreach my $child ($root->children)
```

Comment:
I'm finding that this gives me errors when the share directory does not yet exist, so I put the following above this line:
```perl
return unless -d $root;
```

I came across this because I have CI build of [Alien-MuPDF](https://p3rl.org/Alien::MuPDF) with the latest Alien::Build and Dist::Zilla::Plugin::AlienBuild.

Reply:
gah.  You are right.  #126 should fix this.  and [AlienBuild] 0.29 for dzil.